lib/routes.js

```diff
@@ -21,7 +21,14 @@ module.exports = [
         method: 'GET',
         path: '/register',
         handler: function (req, res) {
-            res.redirect('/');
+            res.redirect('https://hacksi20.eventbrite.com');
+        }
+    },
+    {
+        method: 'GET',
+        path: '/register/forms',
+        handler: function (req, res) {
+            res.file('./static/forms/HackSIAssumptionofRiskForm.pdf');
```

Comment:
Instead of just downloading the file, can we toss up a simple page that contains most of the content from the https://hacksi20.eventbrite.com FAQ section? And then links to the download PDF?